tests/test_runs/test_run.py

```diff
@@ -26,18 +26,19 @@ class TestRun(TestBase):
     # less than 1 seconds
 
     def test_tagging(self):
-        runs = openml.runs.list_runs(size=1)
-        run_id = list(runs.keys())[0]
+        runs = openml.runs.list_runs(size=1, output_format="dataframe")
+        assert not runs.empty, "Test server state is incorrect"
+        run_id = runs["run_id"].iloc[0]
         run = openml.runs.get_run(run_id)
         tag = "testing_tag_{}_{}".format(self.id(), time())
-        run_list = openml.runs.list_runs(tag=tag)
+        run_list = openml.runs.list_runs(tag=tag, output_format="dataframe")
```

Comment:
Is the name `run_list` on purpose here? Previously, you changed every `*_list` together with changing the output format.

Reply:
Oversight, I suppose. Fixed!